From 2d5987cc5dd086a2e670909986719ce32d85015b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1=C5=A1=20Hr=C4=8Dka?= <thrcka@redhat.com>
Date: Tue, 30 Sep 2014 15:37:35 +0200
Subject: [PATCH] use upstream test values

---
 test/mjsunit/regress/regress-1122.js | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/test/mjsunit/regress/regress-1122.js b/test/mjsunit/regress/regress-1122.js
index 815511d..da576ba 100644
--- a/test/mjsunit/regress/regress-1122.js
+++ b/test/mjsunit/regress/regress-1122.js
@@ -62,7 +62,6 @@ assertEquals('prefix 16000 suffix',
 assertEquals('prefix undefined suffix',
              function_with_n_params_and_m_args(32000, 10000));
 
-assertThrows("function_with_n_params_and_m_args(35000, 35000)");
-assertThrows("function_with_n_params_and_m_args(100000, 100000)");
-assertThrows("function_with_n_params_and_m_args(35000, 30000)");
-assertThrows("function_with_n_params_and_m_args(30000, 35000)");
+assertThrows("function_with_n_params_and_m_args(66000, 60000)");
+assertThrows("function_with_n_params_and_m_args(60000, 66000)");
+
-- 
1.8.3.1

